packages/mobile/package.json

```diff
@@ -176,6 +176,7 @@
     "eslint": "^6.8.0",
     "jest-fetch-mock": "^2.1.2",
     "metro-react-native-babel-preset": "^0.58.0",
+    "mocha": "8.2.0",
```

Comment:
Detox needs this now?

Reply:
I will revert his along with the detox revert.

Reply:
It looks like I didn't commit all the files related to the detox upgrade anyway. Sorry about the mess! I knew I should not have tried to upgrade detox in the same PR. I regret doing it since now my build is messed up too. Starting with a clean repo and no upgrades to packages.

Reply:
Thanks, could you add a comment about this?

Reply:
Oh this got reverted so no need to worry about this package.